apps/judicial-system/web/src/routes/CourtOfAppeal/Ruling/Ruling.tsx

```diff
@@ -114,6 +114,7 @@ const CourtOfAppealRuling: React.FC<React.PropsWithChildren<unknown>> = () => {
       ],
       workingCase,
       setWorkingCase,
+      false,
```

Comment:
I'm confused.. `setAndSendCaseToServer` only takes three arguments (updates, workingCase and setWorkingCase). Am I missing something?